ui/apps/dashboard/src/app/(dashboard)/env/[environmentSlug]/event-search/SlideOver.tsx

```diff
@@ -0,0 +1,73 @@
+'use client';
```

Comment:
Is there any other difference between this SlideOver and the dev server one apart from setting the isOpen outside, or is this ready to be a shared component?